<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>209</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Component Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="componentComboBox"/>
   </item>
   <item>
    <widget class="QCheckBox" name="forceSmoothCheckBox">
     <property name="toolTip">
      <string>Force geometry to be rendered with averaged normals to give a smooth appearance</string>
     </property>
     <property name="text">
      <string>Force Smooth</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="forceFacetedCheckBox">
     <property name="toolTip">
      <string>Force geometry to be rendered with per-face normals giving a faceted appearance.</string>
     </property>
     <property name="text">
      <string>Force Faceted</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="nativeIndicesCheckBox">
     <property name="toolTip">
      <string>Map the native indices from the model into the rendered result.  This may improve display of generated models where sharp edges have been constructed rather than specified with smoothing groups.</string>
     </property>
     <property name="text">
      <string>Native Indices</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="correctNormalsCheckBox">
     <property name="toolTip">
      <string>Use face adjacency to attempt to correct normals that are pointing the wrong way.  Suitable if dark or invisible faces are occurring on a larger mesh.</string>
     </property>
     <property name="text">
      <string>Correct Normals</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="correctActuteCheckBox">
     <property name="toolTip">
      <string>Use face adjacency to force faces that are at an acute angle to their neighbours into different smoothing groups. Useful for generated models where different sections of geometry have not been placed into smoothing groups.</string>
     </property>
     <property name="text">
      <string>Correct Acute</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showWarningsCheckBox">
     <property name="text">
      <string>Show Warnings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
